app/models/placement_group.rb

```diff
@@ -0,0 +1,113 @@
+class PlacementGroup < ApplicationRecord
+  acts_as_miq_taggable
+  include YAMLImportExportMixin
+  include SupportsFeatureMixin
+  include NewWithTypeStiMixin
+  def self.new(*args, &block)
+    if self == Authentication && (args.empty? || args.first.kind_of?(Hash))
+      args = [{}] if args.empty?
+      h = args.first
+
+      type = h[inheritance_column.to_sym] || h[inheritance_column.to_s]
+      h[inheritance_column.to_sym] = "AuthUseridPassword" if type.nil?
+    end
+
+    super
+  end
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  has_many :vms, :dependent => :nullify
```

Comment:
Oh, you have `has_many :vms` here...why do you have the `has_and_belongs_to_many :vms` above then?